editor/find_in_files.cpp

```diff
@@ -377,6 +415,26 @@ FindInFilesDialog::FindInFilesDialog() {
 		gc->add_child(hbc);
 	}
 
+	Label *includes_label = memnew(Label);
+	includes_label->set_text(TTR("Includes:"));
+	includes_label->set_tooltip_text(TTR("Include the files with the following expressions. Use \",\" to separate."));
```

Comment:
I couldn't get this tooltip to show, I think because Label ignores mouse by default.
Either the mouse filter on the label should be changes, or the tooltip could go on the line edit.
Same with the excludes tooltip.

Reply:
BTW both of these lines should use `TTRC` instead of `TTR`

Reply:
> BTW both of these lines should use `TTRC` instead of `TTR`

You can see around these lines, they are uses `TTR`, or refer to [contributing document](https://docs.godotengine.org/en/stable/contributing/development/editor/editor_style_guide.html).

Reply:
Maybe docs are outdated. `TTR` translates in place. `TTRC` does nothing, but with auto-transplantation system it allows switch language at runtime without editor restart. After editor auto-transplantation was added, using `TTRC` is recommended.

Reply:
> Maybe docs are outdated. `TTR` translates in place. `TTRC` does nothing, but with auto-transplantation system it allows switch language at runtime without editor restart. After editor auto-transplantation was added, using `TTRC` is recommended.

I prefer to keep consistency with around code.
Even though `TTRC` is recommended to use now, to change these should be in an independent pr.